governance.md

```diff
@@ -0,0 +1,137 @@
+
+### Contributors
+Contributors are community members who contribute in concrete ways to the project, most often in the form of data updates, code and/or documentation. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+Contributors have read-only access to source code and therefore can submit changes via pull requests. Contributor pull requests have their contribution reviewed and merged by a [Peer](#Peers) or [Owner](#Owners). Owners and Peers work with Contributors to review their code and prepare it for merging.
+
+Contributors may also review pull requests. This can be helpful, but their approval or disapproval is not decisive for merging or not merging PRs.
+
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for committership by an existing Peer or Owner.
+
+### Peers
+Peers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. Peers are given push/write access to the project’s GitHub repos.
+
+Peers:
+
+- Are expected to work on public branches of their forks and submit pull requests to the master branch.
+- Must submit pull requests for all their changes.
+- May label and close issues.
+- May merge compat data update pull requests only.
+- Have their non-data update work reviewed by [Owners](#Owners) before acceptance into the repository. Non-data pull requests are PRs that change the schema, update project meta-docs, the linter. or other infrastructure changes.
+- Should ask for super-review from other Peers or Owners on PRs that are disruptive or controversial.
+
+To become a Peer:
+
+- One must have shown a willingness and ability to participate in the project as a team player. Typically, a potential Peer will need to show that they have an understanding of and alignment with the project, its objectives, and its strategy.
+- Have contributed a significant amount of work to the project (e.g. in the form of PRs or PR reviews), thereby demonstrating their trustworthiness and commitment to the project.
+
+New Peers can be nominated by any existing Peers. Once they have been nominated, there will be a vote by the Owners.
+
+It is important to recognize that being a Peer is a privilege, not a right. That privilege must be earned and once earned it can be removed by the Owners. However, under normal circumstances committership exists for as long as the Peer wishes to continue engaging with the project. Inactive Peers might be marked as inactive or removed by the Owners and may re-enter when they choose to contribute again.
```

Comment:
"Inactive Peers might be marked as inactive or removed" -> Does it make sense to define somewhere what kind of time period we are talking about? No activity on the project for longer for a few months or more?

Reply:
Added, good idea!